### PR TITLE
Package models

### PR DESCRIPTION
**Pull request by Atlante45**
Reuse ModelUpload to create .fst file and package .fst/.fbx/textures

Right now it just drops everything in a directory, however I want to zip it too at first.
I don't think we want to get zlib back in though, @birarda?

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1609823e2b2c7900749edf87059d7e12ef8b0eda available at: https://gist.github.com/85ca06ccd97462d45169<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.dmg

**Comment by ZappoMan** [comment]
@clement - you need to merge with upstream master. Some changes made by @ctrlaltdavid in an earlier PR change the validation of skeleton models on packaging.

**Comment by hifi-gustavo** [comment]
Build console output for commit: bcfd4b50be2502a5269c475a82cb80da76724f73 available at: https://gist.github.com/85ca06ccd97462d45169<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.dmg

File: interface/src/ModelPackager.cpp [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+                             QString("Could not open FBX file %1").arg(_fbxInfo.filePath()),
+                             QMessageBox::Ok);
+        qWarning() << QString("ModelPackager::loadModel(): Could not open FBX file %1").arg(_fbxInfo.filePath());
+        return false;
+    }
+    qDebug() << "Reading FBX file : " << _fbxInfo.filePath();
+    QByteArray fbxContents = fbx.readAll();
+    _geometry = readFBX(fbxContents, QVariantHash());
+    
+#if 0 /// Temporarily remove this check until CtrlAltDavid can come up with a fix.
```
Comment (ZappoMan) [inline interface/src/ModelPackager.cpp]: looks like a merge conflict here. I believe this code was deleted yesterday in this PR...

https://github.com/highfidelity/hifi/pull/4399

Comment (Atlante45) [inline interface/src/ModelPackager.cpp]: Darn, added it below but forgot to remove it here.
Fixed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 52c0ba817a158c564626fdc54f67a46b794da5b9 available at: https://gist.github.com/85ca06ccd97462d45169<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4416.dmg

**Pull request merged**